catalog/NASA/NASA_GEOS-CF_v2.libsonnet

```diff
@@ -0,0 +1,53 @@
+{
+  description: |||
+    The Goddard Earth Observing System Composition Forecast (GEOS-CF) system is
+    a high-resolution (0.25&deg;) global constituent prediction system from
+    NASA's [Global Modeling and Assimilation Office(GMAO)](
+    https://gmao.gsfc.nasa.gov/).
+
+    GEOS-CF offers a tool for atmospheric chemistry research, with the goal
+    to supplement NASA's broad range of space-based and in-situ observations.
+    GEOS-CF expands on the GEOS weather and aerosol modeling system by
+    introducing the [GEOS-Chem](http://wiki.seas.harvard.edu/geos-chem/)
+    chemistry module to provide hindcasts and 5-days forecasts of atmospheric
+    constituents including ozone (O<sub>3</sub>), carbon monoxide (CO), nitrogen dioxide
+    (NO<sub>2</sub>), sulfur dioxide (SO<sub>2</sub>), coarse particulate matter 
+    (PM<sub>10</sub>), and fine particulate matter (PM<sub>2.5</sub>). The
+    chemistry module integrated in GEOS-CF is identical to the offline GEOS-Chem
+    model and readily benefits from the innovations provided by the GEOS-Chem
+    community.
+
+    In October 2025, a major update to the GEOS-CF system was introduced as 
+    GEOS-CF v2. This change included updates to the GEOS atmospheric general 
+    circulation model and the GEOS-Chem chemical model, replaying meteorology to 
+    [GEOS IT](https://gmao.gsfc.nasa.gov/gmao-products/geos-it/), direct assimilation 
+    of stratospheric profiles and column ozone from NASA Aura OMI and MLS instruments, 
+    and updates to the anthropogenic emission source. Evaluation of these updates is 
+    ongoing; results will be added into this description as they become available.
```

Comment:
Do you have any known limitations that we can mention here? AI Agents are increasingly using these dataset descriptions to reason about dataset suitability, and the more common-sense wisdom we can mention in this text, the better.